pom.xml

```diff
@@ -208,7 +208,7 @@
     <microprofile.health.version>1.0</microprofile.health.version>
     <microprofile.health.impl.version>1.0.0</microprofile.health.impl.version>
     <microprofile.metrics.version>1.1</microprofile.metrics.version>
-    <microprofile.metrics.impl.version>1.0.0</microprofile.metrics.impl.version>
+    <microprofile.metrics.impl.version>1.0.1-SNAPSHOT</microprofile.metrics.impl.version>
```

Comment:
Hi @ivanjunckes I get an error here:
`[ERROR] Failed to execute goal on project tomee-microprofile-webapp: Could not resolve dependencies for project org.apache.tomee:tomee-microprofile-webapp:war:8.0.0-SNAPSHOT: Failure to find org.apache.geronimo:geronimo-metrics:jar:1.0.1-SNAPSHOT in https://repository.apache.org/service/local/repositories/orgapachegeronimo-1067/content/ was cached in the local repository, resolution will not be reattempted until the update interval of Geronimo OpenAPI Staging has elapsed or updates are forced`
The lib is not found in the maven repo.

Reply:
Fixed